src/main/java/io/confluent/connect/jdbc/sink/JdbcDbWriter.java

```diff
@@ -66,7 +66,9 @@ void write(final Collection<SinkRecord> records) throws SQLException {
       }
       buffer.add(record);
     }
-    for (BufferedRecords buffer : bufferByTable.values()) {
+    for (TableId tableId : bufferByTable.keySet()) {
+      BufferedRecords buffer = bufferByTable.get(tableId);
```

Comment:
Minor: If we're going to use the keys and values, it might be better to iterate over the entries:
```
for (Map.Entry<TableId, BufferRecords> entry : bufferByTable.entrySet()) {
  TableId tableId = entry.getKey();
  BufferedRecords buffer = entry.getValue();
```
Thoughts?